lib/metasploit/framework/login_scanner/wordpress_rpc.rb

```diff
@@ -27,14 +22,12 @@ def attempt_login(credential)
           end
 
           begin
-            http_client.connect
 
-            request = http_client.request_cgi(
+            response = send_request({
                 'uri' => uri,
                 'method' => method,
-                'data' => generate_xml_request(credential.public,credential.private)
+                'data' => generate_xml_request(credential.public,credential.private})
             )
```

Comment:
Might want to make it so the { and }  characters here are on their own lines.

Reply:
Actually this will break the code as you have a curly brace in the wrong location.

Reply:
This was already resolved in a later commit. Marking resolved.